### DISCUSSION TITLE
fotos dos contatos

### DISCUSSION THREAD

**Discussion by abpaula77** [dev-pira/site#252] [General]
As fotos dos contatos estão sendo obtidas de um link direto do linkedin, porém aparentemente esse link é temporário.
Verificar possibilidade de armazenar as fotos ou usar de outro local.
Talvez usar o proprio contentful.

**Comment by murilobeltrame** [comment]
Sim, é possível. 
Da pra armazenar no CRM.

**Reply by murilobeltrame** [reply]
A ser endereçado com #262
